common/elasticsearch/client_v7.go

```diff
@@ -42,12 +43,15 @@ type (
 var _ Client = (*clientV7)(nil)
 
 // newClientV7 create a ES client
-func newClientV7(config *Config) (*clientV7, error) {
+func newClientV7(config *Config, logger log.Logger) (*clientV7, error) {
 	options := []elastic.ClientOptionFunc{
 		elastic.SetURL(config.URL.String()),
 		elastic.SetSniff(false),
 		elastic.SetBasicAuth(config.Username, config.Password),
 
+		elastic.SetErrorLog(newErrorLogger(logger)),
+		// elastic.SetInfoLog(newInfoLogger(logger)),
```

Comment:
Remove commented code

Reply:
I want it to be there in case I will want to see info logs. There is not log level in ES. The only way I can control is to set these loggers or not.

Reply:
+1 to `Remove commented code`

Reply:
Added clarifying comment.